### ISSUE TITLE
Allow installation with bower

### ISSUE THREAD

**Issue by julkue** [afeld/bootstrap-toc#17]

**Comment by julkue** [comment]
@afeld What help do you need here? Just create a [bower.json](https://github.com/bower/spec/blob/master/json.md) and run

```
bower register bootstrap-toc https://github.com/afeld/bootstrap-toc.git
```

**Comment by afeld** [comment]
Right—would happily accept a pull request for the bower.json...I just haven't gotten around to it.

**Comment by mirodietiker** [comment]
We were able to load bootstrap-toc through bower until recently. Did you unregister the package?
